Dockerfile

```diff
@@ -1,52 +1,38 @@
 # Install dependencies only when needed
-FROM node:14.18.1-alpine AS deps
+FROM node:14.18.1-slim AS builder
 
 # Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
-RUN apk add --no-cache libc6-compat
+RUN apt-get update \
+  && apt-get -y install openssl libc6
 
 WORKDIR /app
-# COPY package.json yarn.lock ./
-# We need to copy everything as long as we're using embedded Keystone
+
 COPY . .
 
 RUN yarn install --frozen-lockfile
 
-# Build source code
-FROM node:14.18.1-alpine AS builder
-WORKDIR /app
-COPY . .
-COPY --from=deps /app/node_modules ./node_modules
-
-# Build the app
 ENV NEXT_TELEMETRY_DISABLED 1
-RUN yarn build
 
-# Install production deps only
-RUN rm -rf node_modules && yarn install --production --ignore-scripts --prefer-offline
+ARG NEXT_PUBLIC_SITE_URL
```

Comment:
Do we need to set these in Github Actions where it will build the container?

Reply:
Yes we'll always want to set them